_extras/partner_policy.md

```diff
@@ -0,0 +1,41 @@
+
+Your annual membership gives gives you a number of slots in our instructor training program.  This is a training for people who want to become instructors with the Carpentries (Software Carpentry and Data Carpentry). The workshop is a mix of lectures and hands-on lessons and is very interactive where trainees practice giving short lessons using approaches learned and implement some of the teaching techniques which we will discuss. This is training about pedagogical approach to teaching, not technical training specific to particular lessons. Trainees do not need any particular technical background, and we will not be teaching our workshop curriculum to trainees. This workshop is based on our constantly revised and updated open sourced curriculum for instructor training (https://swcarpentry.github.io/instructor-training/). 
+
+Our trainings are all offered as two day, online events.  You can view our training calendar here (INSERT LINK WHEN WE HAVE ONE) and sign up for a training that meets your team’s needs. A few points as you select dates:
+
+* We recommend that you sign up for an event during the first six months of your partnership year.  This ensures your trainees have time to complete the training program and start teaching workshops during your membership year.
+* You must sign up for this training at  least four weeks in advance. Events may be cancelled if no member organizations sign up by four weeks in advance.
+* All events run from 9am to 5pm in the time zone listed.  You do not need to be in the same time zone.  However, you do need to accommodate that time zone.  For example, a group from US/Central time may join a training scheduled from 9am to 5pm US/Eastern, but would need to be available from 8am to 4pm their time.
+* The number of seats your organization receives depends on your membership level (Silver/Gold/Platinum).  At least six participants must attend at once.  Any unused seats will be forfeited. This is to ensure we can accommodate everyone across all our partner sites. Our  trainings include group exercises that  your  team must work on together.  We also want our training cohorts to recognize they are part of a team which represents the Carpentries at their institution.
+* Again, because we have group exercises, the entire team must be in the same room.  Individuals can not participate remotely.  
+* If an individual misses more than one hour of the training over both days, he/she will not pass the training requirement and his/her spot will be forfeited.  We are training people to deliver Carpentries training and be ambassadors of the Carpentries program, and need to ensure they get our full experience.
```

Comment:
Does this mean that they can lose 1 hour each day, or 30 minutes each day, or 1 hour total over the 2 days, or...?